games-emulation/virtualjaguar/virtualjaguar-2.1.2.ebuild

```diff
@@ -18,8 +18,8 @@ RDEPEND="media-libs/libsdl[joystick,opengl,sound,video]
 	sys-libs/zlib
 	virtual/opengl
 	dev-qt/qtcore:4
-	dev-qt/qtgui:4
-	dev-qt/qtopengl:4
+	dev-qt/qtgui:4[-egl]
+	dev-qt/qtopengl:4[-egl]
```

Comment:
The question is, does `dev-qt/qtgui` with `egl` enabled, trigger some problems? Or does `games-emulation/virtualjaguar` just make no use of `egl` in `dev-qt/qtgui`.